lib/cn/cn_perfc.c

```diff
@@ -14,8 +14,6 @@
 #include "cn_internal.h"
 #include "cn_perfc_internal.h"
 
-#define STATIC
```

Comment:
Nice

Reply:
This is where I would use Tristan's  hse_static macro.